### DISCUSSION TITLE
LBB question

### DISCUSSION THREAD

**Discussion by dimasrepo** [teamalgoritma/community#886] [Q&A] [answered]
Dear Team Algoritma,

Mau sedikit bertanya kalau syntax atau cara membuat orderlist seperti gambar berikut bagaimana ya ? karena sudah coba chek sebelumnya di materi materi  dan di Markdown Cheatsheet ada syntaxnya tapi saya tidak tau cara penggunaannya.
Mohon bantuannya 

Terimakasih 

![image](https://github.com/teamalgoritma/community/assets/141438958/b1a157c9-0a34-4075-92d8-0c5af91a72a0)

**Comment by Yusraf002** [comment, marked as answer]
Dear @senddimasadit 

Terimakasih telah mengajukan pertanyaan melalui forum ini. Mengenai pertanyaan bapak terkait order list, dikelas kita mengenalkannya sebagai _table of contents_. Yang mana dapat bapak atur di bagian YAML pada file Rmd. seperti berikut:
```{r}
---
title: "Habits"
output:
  html_document:
    toc: true
    toc_depth: 2
---
```
 Untuk panduan di penggunaan/pengaturan YAML sudah kami lampirkan pada panduan LBB P4DS-PS. 

Berikut adalah referensi untuk mengatur format report pada R Markdown:
- **YAML R Markdown**: https://bookdown.org/yihui/rmarkdown/html-document.html#table-of-contents
- Themes R Markdown: https://www.datadreaming.org/post/r-markdown-theme-gallery/
- Contoh implementasi Themes R Markdown: https://rpubs.com/ranydc/rmarkdown_themes
- R Markdown Cheatsheet: https://raw.githubusercontent.com/rstudio/cheatsheets/master/rmarkdown-2.0.pdf
- **Tips R Markdown Reporting**: https://holtzy.github.io/Pimp-my-rmd/

Demikian jawaban yang dapat kami berikan, bisa bapak pelajari terlebih dahul. Apablia ada pertanyaan lebih lanjut bisa kita lanjutkan diskusi pada chat ini. dan Apabila jawaban diatas sudah menjawab pertanyaan bapak boleh **_mark as answer_** ya pak.

Best Regards,
Team Algoritma.

**Reply by IrfanChairurrachman** [reply]
Halo @dimasrepo  , untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
